src/shared/ui/pickerPrompter.ts

```diff
@@ -49,10 +53,24 @@ export type ExtendedQuickPickOptions<T> = Omit<
     placeholderItem?: DataQuickPickItem<T>
     /** Item to show if there was an error loading items */
     errorItem?: DataQuickPickItem<T>
+    /**
+     * Appends 'Selected previously' to the last selected item's description text (default: true)
+     * This currently mutates the item as it is expected that callers regenerate items every prompt
+     */
+    showSelectedPreviously?: boolean
+}
+
+const DEFAULT_PLACEHOLDER_ITEM = {
+    label: localize('AWS.picker.dynamic.noItemsFound.label', '[No items found]'),
```

Comment:
Nit: should we turn this into a function so it's easier for us to set something more specific than "items"?

Reply:
There's an option to set this placeholder called `placeholderItem` when creating the prompt
A function could make since too though.